### PR TITLE
Update Onboard grant jam

### PR DESCRIPTION
**Pull request by BrightTheBackpack**
Update instructions as asked by Karmanyaah
See [#344](https://github.com/hackclub/OnBoard/pull/344)

### CODE REVIEW

**Comment by karmanyaahm** [comment]
Can you please add this somewhere in there: https://github.com/hackclub/OnBoard/blob/71480c5fad8356149284f041f209eb5591df6a64/README.md?plain=1#L52-L53

TYSM for updating both of these though!

File: jams/singles/onboard-grant/en-US.md [hunk 1]
```diff
@@ -82,7 +82,8 @@ In all, you should have the following files under your project folder:
 - [ ] `README.md`: A filled out [`TEMPLATE.md`](./projects/!Template/TEMPLATE.md?plain=1), renamed to `README.md`
 - [ ] `cart.png`: A screen shot of your vendor approving your board
 - [ ] `Gerbers.zip`: This should be the file that you send off to the vendor.
-- [ ] Design files (`design.json`, `design.sch`, `design.brd`, etc). Whatever format your designer outputs to should be included.
+- [ ] Design files(`design.json`, `design.sch`, `design.brd`, etc) in `src` folder . Whatever format your designer outputs to should be included.
+- [ ] `Schematic.pdf`: A pdf of your schematic, obtained from you're designer.
```
Comment (karmanyaahm) [inline jams/singles/onboard-grant/en-US.md:86]: file name capitalization + do you think the explicitly listed out file formats in the README would be relevant here?

See https://github.com/hackclub/OnBoard/blob/71480c5fad8356149284f041f209eb5591df6a64/README.md?plain=1#L94-L99

**Comment by karmanyaahm** [comment]
@SerenityUX can you merge this as is please.

**Comment by maxwofford** [comment]
Thank you @BrightTheBackpack!

**Pull request merged**